src/octopeer-github/main/ElementEventBinding/ClickElementEventBinding.ts

```diff
@@ -1,13 +1,11 @@
+/// <reference path="../ElementSelectionBehaviour/ElementSelectionBehaviour.ts"/>
 /**
  * Created by Youri on 04/05/2016.
- */
-
-/**
  * Class for adding click events.
  */
 class ClickElementEventBinding implements ElementEventBinding {
-    private eventType: string = "click";
-    private eventID: EventID  = new EventID(1);
+    public eventType: string = "click";
+    public eventID: EventID  = new EventID(101);
```

Comment:
That's fine, but please note that I will place comments on the document in our drive document :)